src/utils/tool.js

```
@@ -359,4 +359,16 @@ tool.arrSum = (arr) => {
   return sum
 }
 
+tool.uuid = () => {
+  const randomBytes = CryptoJS.lib.WordArray.random(16);
+  const randomHex = randomBytes.toString(CryptoJS.enc.Hex);
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)